app/editor/src/components/form/select/styled/SelectField.tsx

```diff
@@ -122,6 +122,26 @@ export const SelectField = styled(Select)<ISelectProps<any>>`
 
     &:focus {
       outline: 0;
+      box-shadow: ${(props) => {
```

Comment:
This regrettably doesn't work because the react-select has a different structure.  We'll need to figure it out somehow.  I don't want to use the `customStyle` inline style option.